app/src/main/res/drawable/ic_star.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Cool. I don't know much about designs. This shoulds good to me. 
@kunall17 Any suggestions?